### ISSUE TITLE
Can you explain the difference between this and the other starter?

### ISSUE THREAD

**Issue by naivefun** [michaeljota/angular2-ultimate-starter#1]
I see the electron support and feel keen to use this to replace https://github.com/AngularClass/angular2-webpack-starter

But I'm not sure what I would miss if I make the switch. Can you give some details? Thanks.

**Comment by michaeljota** [comment]
This is mainly based on that one. In addition to everything you get from that source, you get:

- ngrx/store support, and examples.
- Electron support, with TypeScript and webpacked.
- Sass support, if you use this.
- Webfonts support, loaded inline as a string.
- A different file structure.

I think you won't miss anything, cause this extends from there. Maybe only the HMR, that I did not like, and found it incompatible with the ngrx/store. It may get back if I found another implementation. 

BTW: If you don't know what HMR is, then you won't miss it. It suppose to reload the application keeping the state, but that implementation 'just works'. I rather use the ngrx/store, to manage the state.
